### PR TITLE
style: SIP-34 control-label

### PR DESCRIPTION
**Pull request by mistercrunch**
# After
Screen Shot 2020-07-21 at 12 53 30 PM
Screen Shot 2020-07-21 at 12 53 19 PM
Screen Shot 2020-07-21 at 12 52 38 PM
Screen Shot 2020-07-21 at 12 52 10 PM
Screen Shot 2020-07-21 at 12 51 37 PM

# Before
Screen Shot 2020-07-21 at 12 57 21 PM
Screen Shot 2020-07-21 at 12 57 13 PM
Screen Shot 2020-07-21 at 12 57 03 PM
Screen Shot 2020-07-21 at 12 56 51 PM

### CODE REVIEW

File: superset-frontend/stylesheets/less/cosmo/variables.less [hunk 1]
```diff
@@ -220,7 +220,7 @@
 );
 
 // ** `.form-group` margin
-@form-group-margin-bottom: 10px;
+@form-group-margin-bottom: 16px;
```
Comment (mistercrunch) [inline superset-frontend/stylesheets/less/cosmo/variables.less:223]: moving towards the `8px` system

**Review by nytai — APPROVED** [review]
lgtm

File: superset-frontend/stylesheets/less/cosmo/variables.less (hunk 1 above)
Comment (rusackas) [reply superset-frontend/stylesheets/less/cosmo/variables.less:223]: I might sweep this (and others) up with a @grid-unit variable... no need to worry for this PR though.

**Review by rusackas — APPROVED** [review]

**Comment by rusackas** [comment]
Impacts #8976

**Pull request merged**